packages/documentation/src/pages/getting-started/common-tasks/workflow/deploy.mdx

```diff
@@ -0,0 +1,90 @@
+
+The **use of hotfixes is discouraged**, but may be useful in an emergency situation when `master` has significantly deviated from the release and a fix to the failed production release is critical. To create a hotfix, create a branch from the last stable release tag, make changes necessary (with review), create a new release tag following the correct naming scheme, and trigger a deploy in Jenkins with the release name as a parameter.
+
+If SLOs are not affected and a fix is not critical, no rollback will be issued. Instead the fix should be applied through the standard development workflow.
+
+## Manual deployment
+
+Two out-of-cycle deploys are supported in Jenkins:
+- partial deploy including only static page changes (`vagov-content` and `Drupal`)
+- full deploy of VA.gov client app and static pages
+
+### Before deploying
+
+- Wait for **Jenkins** to **build the change** in `vets-website`
+- Builds status can be viewed [here](http://jenkins.vetsgov-internal/blue/organizations/jenkins/testing%2Fvets-website/activity?branch=master). _Requires SOCKS proxy. See [Accessing internal tools](/getting-started/internal-tools)_
+- If this build fails, you may need to log into Jenkins and restart it
+
+### Partial deploy- static page changes only
+
+1. Start a deploy job
+   - Log into Jenkins [here](http://jenkins.vetsgov-internal/job/deploys/job/vets-gov-autodeploy-vets-website/)
+   - Click **Build with Parameters**
+   - Set the **release_wait** option to **5** minutes
+   - Check **use_latest_release** <-- _important_
+   - Click **Build**
+   - Verify commits in **deployment notification**
+
+ _In Slack channel **#vetsgov-engineers**, Jenkins will include a link that shows the exact commits being released in the **deploy notification**._
+
+2. **Verify** changes in **production** once the build finishes
+
+### Full deploy of VA.gov client app
+
+1. Verify that your changes are committed and that the changes since the last deploy are safe to deploy:
+   - **Last deployment time**: You can check the build [history](http://jenkins.vetsgov-internal/job/deploys/job/vets-gov-autodeploy-vets-website/) for the time of the last deploy (usually daily at 2pm EST)
+   - **Commit history**: [Look](https://github.com/department-of-veterans-affairs/vets-website/commits/master) for commits after the last deploy and verify they're production ready.
+
+_You may need to contact the developers of those commits to verify._
+
+2. Start a deploy job
+   - Log into Jenkins [here](http://jenkins.vetsgov-internal/job/deploys/job/vets-gov-autodeploy-vets-website/)
+   - Click **Build with Parameters**
+   - Set the **release_wait** option to **5** minutes
+   - Uncheck **use_latest_release** <-- _important_
```

Comment:
Since this is the main difference between the two processes, I decided to to include an "Uncheck" step in the full deploy description.